### DISCUSSION TITLE
Uploading CSS

### DISCUSSION THREAD

**Discussion by nicogithub22** [globaleaks/globaleaks-whistleblowing-software#3358] [General]
Hello,
i try to upload a style.css, but after uploading nothing changed? Why? Does the stylesheet have to be in a certain format?

Thanks,
Nico

**Comment by evilaliv3** [comment]
Thank you @nicogithub22 for your questions.

We are currently not aware of bugs on this functionalities.

Is it possible that you are loading a CSS on a secondary tenant? 
In this case the CSS will be visible only when loading the secondary tenant directly but not when you administer the tenant via the management tenant (also called root tenant).

**Comment by nicogithub22** [comment]
Thank you for your quick answer
I have tested it in two different environments.
1. creating a style.css file
2. content: 
body {
    background-color: aqua;
    }
3. upload via Admin GUI
4. reload page(not admin-site) and check appearance
5. no change

**Comment by evilaliv3** [comment]
Thank you for reporting this.

GlobaLeaks support both ltr and rtl orientations so that when writing your css you should specify to which orientation it applies.

In this case you could write the following, to state that you configuration applies to both the two orientations

```css
[dir] body {
  background-color: aqua;
}
```

**Comment by nicogithub22** [comment]
Thanks that worked!
I have one more question.
How can I now integrate this font?
@font-face {
    font-family: 'Roboto';
    src: url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@1,100&display=swap') format('truetype');
    font-weight: normal;
    font-style: normal;
    }
[dir] body {
    font-family: 'Roboto';
    font-size: 16px;
}
Thats does not work

**Comment by evilaliv3** [comment]
Thank you for your question.

If possible please always divide your tickets by topic or question.

In relation to the question that you just made, GlobaLeaks for security
reasons implements a content security policy that block access to third
party resources and this is the reason why you are not able to load fonts
from Google.

In order to load a custom font, load the font on GlobaLeaks as a file
customization and then use it in your css file.

Files loaded on the platform will be accessible on the path /s/

In relation to fonts we advice to use the WOFF format that at the moment is
the one that benefit of good compression and great compatibility with all
the browsers

**Comment by nicogithub22** [comment]
Okay, thank you it works!
